src/bl.c

```diff
@@ -63,15 +64,24 @@ void bl_init(void)
 	bl_usb_init();
 	bl_led_init();
 	bl_acq_init(rcc_config->ahb_frequency);
+#if defined(BL_SPI_MOTHER) || defined(BL_SPI_DAUGHTER)
+	bl_spi_init();
+#endif
 }
 
+#ifndef BL_SPI_TEST
 int main(void)
 {
 	bl_init();
 
 	bl_led_status_set(true);
 
 	while (true) {
+#ifdef BL_SPI_DAUGHTER
+		bl_spi_slave_poll();
```

Comment:
I'd use the same terminology everywhere.

```suggestion
		bl_spi_daughter_poll();
```

Reply:
+1

the BL_SPI_XXX environment will be removed in new patches, as we move to auto detect and only build a single firmware for both master/slave.
I'll make sure the terminology is consistent in new series